cwl/src/main/scala/cwl/CommandOutputBinding.scala

```diff
@@ -220,7 +220,13 @@ object CommandOutputBinding {
         val globs = List(cwlPath)
         globs traverse loadFileWithContents(ioFunctionSet, commandOutputBinding)
       case WomMaybePopulatedFileType =>
-        val globs = Await.result(ioFunctionSet.glob(cwlPath), Duration.Inf)
+        val globs: Seq[String] = Await.result(ioFunctionSet.glob(cwlPath), Duration.Inf).
+
+        //TODO: HACK ALERT - DB: I am starting on ticket https://github.com/broadinstitute/cromwell/issues/3472 which will redeem me of this mortal sin.
+          filterNot{s =>
+            s.endsWith("rc.tmp") || s.endsWith("docker_cid") || s.endsWith("script") || s.endsWith("script.background") || s.endsWith("script.submit") || s.endsWith("stderr") || s.endsWith("stderr.background") || s.endsWith("stdout") || s.endsWith("stdout.background")
```

Comment:
I did in fact push this code on purpose to see if it causes damage or even passes the detritus test.  It got me over the hump on GDC bamfastq_align. I am starting ticket #3092 to provide an actual solution.